dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryActions.java

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.microprofile.telemetry.internal_fat;
+
+import java.util.Arrays;
+import java.util.List;
+
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.rules.repeater.FeatureSet;
+import componenttest.rules.repeater.MicroProfileActions;
+import componenttest.rules.repeater.RepeatActions;
+import componenttest.rules.repeater.RepeatTests;
+
+public class TelemetryActions {
+    public static final String MP14_MPTEL11_ID = MicroProfileActions.MP14_ID + "_MPTEL11";
+    public static final String MP41_MPTEL11_ID = MicroProfileActions.MP41_ID + "_MPTEL11";
+    public static final String MP50_MPTEL11_ID = MicroProfileActions.MP50_ID + "_MPTEL11";
+
+    public static final FeatureSet MP14_MPTEL11 = MicroProfileActions.MP14
+                    .addFeature("mpTelemetry-1.1")
+                    .build(MP14_MPTEL11_ID);
+
+    public static final FeatureSet MP41_MPTEL11 = MicroProfileActions.MP41
+                    .addFeature("mpTelemetry-1.1")
+                    .build(MP41_MPTEL11_ID);
+
+    public static final FeatureSet MP50_MPTEL11 = MicroProfileActions.MP50
+                    .addFeature("mpTelemetry-1.1")
+                    .build(MP50_MPTEL11_ID);
+
+    //All MicroProfile Telemetry FeatureSets - must be descending order
+    private static final FeatureSet[] ALL_MPTEL_SETS_ARRAY = { MP14_MPTEL11, MP41_MPTEL11, MP50_MPTEL11 };
+    private static final List<FeatureSet> ALL_MPTEL_SETS_LIST = Arrays.asList(ALL_MPTEL_SETS_ARRAY);
```

Comment:
Referring back to my comment about not mixing TelemetryActions with MicroProfileActions. What you should do in future is to include all the FeatureSets that you want to use here, in TelemetryActions. You can still make use of some of the constants from MicroProfileActions but the FeatureSets should be included in the "ALL" set here.

Reply:
I would love for this to be rolled into MicroProfileActions with an option to have mpTelemetry 1.1 as a standalone feature for MP 1.4, 4.1 and 5.0.

Reply:
@jhanders34 I'm not entirely sure what you mean by this.

Would you add `mpTelemetry-1.1` to the list of features for MP 1.4, MP 4.1 and MP 5.0? Or were you suggesting moving the `MP14_MPTEL11`, `MP41_MPTEL11` and `MP50_MPTEL11` feature sets to `MicroProfileActions`? Or just adding `mpTelemetry-1.1` to the standalone feature lists? Or something else?